worker/include/RTC/RtpDictionaries.hpp

```diff
@@ -20,6 +20,7 @@ namespace RTC
 		};
 	};
 
+	// NOLINTNEXTLINE (cppcoreguidelines-pro-type-member-init)
```

Comment:
What is clang-tidy complaining about here about?

Reply:
https://clang.llvm.org/extra/clang-tidy/checks/cppcoreguidelines/pro-type-member-init.html

The check flags user-provided constructor definitions that do not initialize all fields that would be left in an undefined state by default construction.

In this case `type` and `subType` are unset on construction. We later on call `SetMimeType()` which gives value to those.